db-scripts/createdb.sql

```diff
@@ -31,6 +31,15 @@ create table if not exists users(
    foreign key(major) references majors(name) on update cascade
 );
 
+create table if not exists session(
+   id serial,
```

Comment:
according to lucia docs session id must be text, not integer

Reply:
done